admin/services/data-access/group.data.service.js

```diff
@@ -57,6 +57,28 @@ groupDataService.sqlFindOneById = async (groupId, schoolId) => {
   return R.head(result)
 }
 
+/**
+ * Get group by school id.
+ * @param schoolId
+ * @returns {Promise<void>}
+ */
+groupDataService.sqlFindOneBySchoolId = async (schoolId) => {
```

Comment:
@GeorgeChatzigiannis you may find more than one group with this query, so maybe `sqlFindOne...` is misleading